Sources/BraintreePaymentFlow/BTPaymentFlowClient.swift

```diff
@@ -20,23 +16,37 @@ import BraintreeCore
     private var paymentFlowName: String {
         return paymentFlowRequestDelegate?.paymentFlowName() ?? "local-payments"
     }
-    
+    public var returnedToAppAfterPermissionAlert: Bool = false
+
     // MARK: - Public Methods
     
+    var webAuthenticationSession: WebAuthenticationSession
```

Comment:
This looks to be internal, is there a reason it's under the `Public Methods` mark?

Reply:
you are right, we only need it as public if we move WebAuthenticationSession to Core module, I think,